uiautomationutilities/build.gradle

```diff
@@ -47,7 +60,6 @@ android {
 
 
     defaultConfig {
-        multiDexEnabled true
```

Comment:
No longer needed?

Reply:
doesn't seem to be?  If it is, isn't it mainly important for desugaring?

Reply:
multiDex (as I understand it) is usually a workaround for method counts exceeding the apk dex file limit

Reply:
Might be needed for UI Automator I think

Reply:
I just put it back.  It shouldn't hurt anything anyway.